docs/core/extensions/sslstream-best-practices.md

```diff
@@ -0,0 +1,107 @@
+---
+title: TLS/SSL best practices
+description: Learn best practices to using SslStream in .NET
+author: rzikm
+ms.author: rzikm
+ms.date: 10/25/2022
+---
+
+# TLS/SSL best practices
+
+TLS (Transport Layer Security) is a cryptographic protocol designed to secure communication between two computers over the internet. The TLS protocol is exposed in .NET via the <xref:System.Net.Security.SslStream> class.
+
+This article presents best practices for setting up secure communication between client and server and assumes .NET Core version 3.1 or later. For best practices for .NET Framework, see [Transport Layer Security (TLS) best practices with the .NET Framework](/dotnet/framework/network-programming/tls).
```

Comment:
3.1 is out of support as off today. While that does not matter for the recommendation I would probably not specify any version or go back all the way to 2.x. Starting with 6, we no longer call it "Core" and that makes it even more confusing IMHO.